Formula/nsc.rb

```diff
@@ -8,6 +8,14 @@ class Nsc < Formula
   version "2.7.1"
 
   on_macos do
+    if Hardware::CPU.intel?
+      url "https://github.com/nats-io/nsc/releases/download/2.7.1/nsc-darwin-amd64.zip"
```

Comment:
Adding note that we eventually want to add the package version in the zip file as well: 
https://github.com/nats-io/nsc/pull/472